packages/relay/src/lib/eth.ts

```diff
@@ -1211,6 +1211,25 @@ export class EthImpl implements Eth {
       ]);
       if (result) {
         if (result?.type === constants.TYPE_TOKEN) {
+          if (blockNumber && !this.common.blockTagIsLatestOrPending(blockNumber)) {
```

Comment:
what happens if `blockNumber` is `earliest`? https://ethereum.org/en/developers/docs/apis/json-rpc/#default-block

Reply:
It was passing through the if, but the parseInt was returning "234" which is why it slipped through my testing. Good catch!